test/Microsoft.AspNetCore.Mvc.Abstractions.Test/ModelBinding/ModelMetadataTest.cs

```diff
@@ -302,6 +302,36 @@ public void GetDisplayName_ReturnsTypeName_WhenPropertyNameAndDisplayNameAreNull
             Assert.Equal("String", result);
         }
 
+        [Fact]
+        public void GetContainerMetadata_ByDefault_ReturnsNull()
```

Comment:
Rename test. This isn't about defaults; it's about metadata for types versus metadata for properties.